masterlist.yaml

```
@@ -18777,24 +18778,38 @@ plugins:
   # Main Mod #
   - name: 'UltimateCollege.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/44797' ]
+    inc:
+      - 'OCW_LightingPatch_Brighter.esp'
+      - 'OCW_LightingPatch_Darker.esp'
+      - 'MagicalCollegeofWinterhold.esp'
+      - 'CollegeModifications.esp'
```

```suggestion
      - 'CollegeModifications.esp'
      - 'MagicalCollegeofWinterhold.esp'
      - 'OCW_LightingPatch_Brighter.esp'
      - 'OCW_LightingPatch_Darker.esp'
```
Not crucial, but sorting these alphabetically would be appreciated.